code/js/rcalc-math.js

```diff
@@ -0,0 +1,56 @@
+var isDivisionPerformed = false;
+
+// displays numerous inputs (numbers) on the screen
+function displayNumber(inputValue) {
+  nameCalculator.nameDisplayMain.value = nameCalculator.nameDisplayMain.value + inputValue;
+}
+
+// testing 0 input, when 0 comes after / buttons are disabled and message is given
+function testZero(inputValue) {
+  if (isDivisionPerformed && inputValue == 0) {
+    var buttonDisable = document.getElementsByClassName("rcalc-button");
+    for (var i = 0; i < buttonDisable.length; i++) {
+      buttonDisable[i].disabled = true;
+      buttonDisable[i].style.color = "black";
+      document.getElementById("id-display-info").value = "Can not devide by 0! Click C";
+    }
+  } else {
+    isDivisionPerformed = false;
+  }
+}
+
+// sets warning, division is in play, boolean used
+function setWarning() {
+  isDivisionPerformed = true;
+}
+
+// clears both displays, resets boolean, enables buttons
+function clearDisplay() {
+  isDivisionPerformed = false;
+  document.getElementById("id-display-main").value = "";
+  document.getElementById("id-display-info").value = "";
+  var buttonEnable = document.getElementsByClassName("rcalc-button");
+  for (var i = 0; i < buttonEnable.length; i++) {
+    buttonEnable[i].disabled = false;
+    buttonEnable[i].style.color = "white";
+  }
+}
+
+function result() {
```

Comment:
result()   ... result what? .... better naming what happens in the method
... please split method to two methods ... setButtonDisable() ... setValuesInDisplay() ... or what you want but method is too big and it is possible to split it to independent parts.

Reply:
result () renamed to evaluateDisplay() also this method is splited